apps/website/src/app/layout.tsx

```diff
@@ -30,9 +31,21 @@ export default function RootLayout({ children }: { children: React.ReactNode })
         <html lang="en" suppressHydrationWarning>
             <body suppressHydrationWarning>
                 <Providers>
+                    <Banner>
+                        Semaphore V4
+                        <Link
```

Comment:
The hover effect on links should be the same as the other links on the website. I think there should just be a change in the underline. You can check the other links.

Reply:
you mean, just removing the `underline` text decoration?

Reply:
updated!

Reply:
Great! It's ready then